cmd/build.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	esbuild "github.com/evanw/esbuild/pkg/api"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildCmd = &cobra.Command{
+		Use:   "build",
+		Short: "Ping the StormForger API",
+		Long:  `Ping the StormForger API and try to authenticate.`,
+		Run:   runBuildCmd,
+	}
+
+	buildOpts struct {
+		Replacements []string
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(buildCmd)
+
+	buildCmd.PersistentFlags().StringArrayVar(&buildOpts.Replacements, "define", []string{}, "Substitute a list of K=V while parsing: debug=false")
```

Comment:
Take a look at https://godoc.org/github.com/spf13/pflag#StringToStringVar

Reply:
I did not got a value passed in containing quotes properly.

Using

```golang
buildCmd.PersistentFlags().StringToStringVar(&buildOpts.Replacements, "define", nil, "Substitute a list of K=V while parsing: debug=false")
```

With `--define='replacements.ENV="production"'` seems to drop the last double quotes. I've also tried different variations of escaping.

Any idea?

Reply:
I think that is on purpose as it the way to quote the value. If we want to keep the quote, we maybe go back to do it manually. 😞